### PR TITLE
Added a border in the pricing page for lg+ screens

### LINKED ISSUE #9337: [DOCS] Enhancing the Pricing Page
### Description

We can add that white border in desktop screens too, it will help the user to easily spot the differences b/w the two plans.

### Screenshots

image
image

### Do you want to work on this issue?

Yes

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Hero-codes**
Edit:
Closes #9337 

## Changes proposed

Added a border in the pricing page for lg+ screens, it enhances the UI.

Before: 
image

After:
image

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by waellerbe** [comment]
Excellent example to learn from

File: pages/pricing.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default function Premium({ user }) {
           }
           return false;
         },
-        onClick: () => {},
+        onClick: () => { },
```
Comment (akash19coder) [inline pages/pricing.js:66]: Please remove the space in between those curly braces and keep it as it was before.

File: pages/pricing.js [hunk 2]
```diff
@@ -386,7 +386,7 @@ export default function Premium({ user }) {
                               <span>
                                 {feature.name}{" "}
                                 {typeof feature.tiers[tier.name] ===
-                                "string" ? (
+                                  "string" ? (
```
Comment (akash19coder) [inline pages/pricing.js:389]: also please remove that extra space that you have added before "string" and make it as it was before

File: pages/pricing.js [hunk 3]
```diff
@@ -412,12 +412,11 @@ export default function Premium({ user }) {
                     className="flex w-1/3 px-4"
                     aria-hidden="true"
                     style={{
-                      marginLeft: `${
-                        (tiers.findIndex((tier) => tier.mostPopular) + 1) * 33
-                      }%`,
+                      marginLeft: `${(tiers.findIndex((tier) => tier.mostPopular) + 1) * 33
```
Comment (akash19coder) [inline pages/pricing.js:415]: I don't see any changes in these line, however the indentation looks disturbed compared to original state. Please format your document(Ctrl + Shift + I in VS Code) before pushing changes.

**Review by akash19coder — CHANGES_REQUESTED** [review]

**Comment by akash19coder** [comment]
How does the changes look in light mode? Could you please share the screenshots of light mode as well?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution.

Please remove the changes to the lock file, as the package file has not been updated. You can avoid these mistakes by:
- checking the files change and only committing the changes you want
- also using `npm ci` to avoid phantom bugs locally and also generating a new lock file

**Comment by SaraJaoude** [comment]
As there has been no activity I am closing this issue.

**Pull request closed without merging**